test/test_quantized.py

```diff
@@ -247,6 +247,72 @@ def test_qfc(self):
         # Assert equal
         np.testing.assert_equal(Y_q_ref, Y_q.int_repr().numpy())
 
+    def test_qfcrelu(self):
+        qfc_prepack = torch.ops.quantized.fbgemm_linear_prepack
+        qfcrelu = torch.ops.quantized.fbgemm_linear_relu
+
+        batch_size = 4
+        input_channels = 16
+        output_channels = 8
+
+        X_scale = 1.5
+        X_zp = 5
+        X_value_min = 0
+        X_value_max = 225
+        X_q0 = np.round(
+            np.random.rand(batch_size, input_channels) * (X_value_max - X_value_min)
+            + X_value_min
+        ).astype(np.uint8)
+
+        W_scale = 0.4
+        W_zp = 2
+        W_value_min = -128
+        W_value_max = 127
+        W_q0 = np.round(
+            np.random.rand(output_channels, input_channels)
+            * (W_value_max - W_value_min)
+            + W_value_min
+        ).astype(np.int8)
+
+        avoid_vpmaddubsw_overflow_fc(
+            batch_size,
+            input_channels,
+            output_channels,
+            X_q0,
+            X_value_min,
+            X_value_max,
+            W_q0,
+            W_value_min,
+            W_value_max,
+        )
+
+        X = torch.from_numpy(_dequantize(X_q0, X_scale, X_zp)).to(dtype=torch.float)
+        W = torch.from_numpy(_dequantize(W_q0, W_scale, W_zp)).to(dtype=torch.float)
+
+        X_q = X.quantize_linear(scale=X_scale, zero_point=X_zp)
+        W_q = W.quantize_linear(scale=W_scale, zero_point=W_zp + 128)
+        b_q = torch.round(torch.rand(output_channels) * 10 - 10).to(dtype=torch.int32)
+
+        # Make sure X_scale * W_scale * input_channels * X_value_max * W_value_max
+        # matches Y_scale * 255 (max for uint8).
+        Y_scale = 1235.1234
```

Comment:
Similar request as before: Please use a different scale and zero-point for y.

Reply:
If we assume all 0 zero-points for X, W, and Y, then the max value for Y will be X_scale * W_scale (X_value_max * W_value_max * 16) = 310896. If this number is much larger than Y_scale * 255, we will get lots of 255 in the result matrices. Then the comparisons in the unit test will not make too much sense.